velox/docs/functions/spark/datetime.rst

```diff
@@ -66,10 +66,20 @@ These functions support TIMESTAMP and DATE input types.
         SELECT dayofweek('2009-07-30'); -- 5
         SELECT dayofweek('2023-08-22 11:23:00.100'); -- 3
 
-.. function:: dow(x) -> integer
+.. spark::function:: dow(x) -> integer
 
     This is an alias for :func:`day_of_week`.
 
+.. spark::function::from_unixtime(unixTime, format) -> string
```

Comment:
Can we document all the supported format patterns?

Reply:
Just gave some valid examples. It may be unnecessary to provide an exhaustive list. Thanks!